.gitlab-ci.yml

```diff
@@ -1,126 +1,21 @@
 stages:
-    - empty
+    - build-test
 
-dummy:
-  stage: empty
+trigger-simnet-for-build-and-test:
+  stage:                     build-test
+  variables: 
+    # The branch or tag name for which project is built. 
+    TRGR_REF:                "${CI_COMMIT_REF_NAME}"
+  image:                     "docker.io/paritytech/tools:latest"
   script:
-      - echo nothing to see here
+    - echo 'TRGR_REF is ${TRGR_REF}'
+    - ./scripts/trigger_pipeline.sh
```

Comment:
bear in mind that it's already the third place you have to maintain this script

Reply:
hopefully we will get rid of 2 places once we move tests to substrate and polkadot repos

Reply:
the idea was to move this script to the simnet side and just pass variables to it

Reply:
but I remember reading you made the first version of this script because there was a bug in gitlab and the triggering was not working properly ... was not this? 

I could place this script in simnet/scripts and use wget to use in the pipeline of each simnet component,
to remove the

Reply:
The bug was about getting a status from a triggered downstream pipeline.
On the last meeting we've discussed how to make it easier and have decided to move stuff around. Namely this script and a part of simnet CI were to merge on the simnet side, and every project would just run it passing flags of variables. That will save you from having to maintain it everywhere and would simplify the launch of the simnet.
The mean of launching it must be in the project.

Reply:
perfect, then why do you need this one? Just do one step forward and
- in case you run a CI to update the simnet codebase you trigger the simnet CI
- in case you use stable simnet from polkadot, avoid the CI triggering and run it directly

Reply:
yes, this is the plan; I wanted to have CI for components of simnet, in case devs want to add more features;
more devs started using gurke and started adding PRs. I wanted to have a CI working for simnet components before moving to next version of simnet